Volume/Views/MainView and Tabs/ReadsView.swift

```diff
@@ -58,10 +58,16 @@ struct ReadsView: View {
                     
                     Spacer()
                     
-                    hamburgerMenu
+                    HamburgerMenuView()
                         .onTapGesture {
                             Haptics.shared.play(.light)
+                            withAnimation(.spring()) {
+                                showPublication.toggle()
+                            }
                         }
+                    
+                    Spacer()
```

Comment:
not too sure about swiftui's styling but i saw you wrapped your previous `Spacer` from line 58-60 with linebreaks so i'm not sure if there should be some here as well..?

Reply:
Usually, when you introduce any View in SwiftUI, you want to separate it with line breaks. `Spacer()` is a View and separating it from the rest of the code makes it stand out. In SwiftUI, it's important to know where your Views are because everything is a View!!